### PR TITLE
Adding a fixture

### PR DESCRIPTION
**Pull request by NotionCommotion**
Please see https://github.com/rectorphp/rector/issues/6563.  Sorry in advance if I am not doing this correctly.  Thanks

### CODE REVIEW

File: app/SomeClass.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    public function __construct()
+    {
+        $this->count = 123;
+    }
+
+    public function getCount():int
+    {
+        return $this->count;
+    }
+}
```
Comment (TomasVotruba) [inline app/SomeClass.php]: This file is probably left-over from local testing, should be removed

File: rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/michael_test.php.inc [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?php
+// rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/michael_test.php.inc
+
```
Comment (TomasVotruba) [inline rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/michael_test.php.inc]: This line is compared to half after `-----`, so it should be removed as it's missing in 2nd half.

**Comment by TomasVotruba** [comment]
It looks good :+1: I've added 2 details

File: app/SomeClass.php (hunk 1 above)
Comment (NotionCommotion) [reply app/SomeClass.php]: Didn't realize, however, that I should remove the app testing code and will do so going forward.

**Comment by NotionCommotion** [comment]
Still don't understand why I get inconsistent results as described by https://github.com/rectorphp/rector/issues/6563#issuecomment-877829870, and how this fixture would shed any light.

I am assuming that you don't want this test since it is already covered by https://github.com/rectorphp/rector-src/blob/main/rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/property.php.inc, right?  Should I close it or you?

Thanks!

File: rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/michael_test.php.inc (hunk 2 above)
Comment (TomasVotruba) [reply rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/michael_test.php.inc]: This line is still breakin the test

**Comment by TomasVotruba** [comment]
The fixture is correct and now we have a clear confirmation.

Merging to move forward :+1:

**Pull request merged**